packages/ngtools/webpack/src/gather_diagnostics.ts

```diff
@@ -56,29 +56,29 @@ export function gatherDiagnostics(
     const tsProgram = program as ts.Program;
     // Check syntactic diagnostics.
     time(`${benchmarkLabel}.gatherDiagnostics.ts.getSyntacticDiagnostics`);
-    checkDiagnostics(tsProgram.getSyntacticDiagnostics);
+    checkDiagnostics(tsProgram.getSyntacticDiagnostics.bind(tsProgram));
```

Comment:
`() => tsProgram.getSyntacticDiagnostics()` ?

Reply:
Yeah that looks better imho.